### PR TITLE
Password match utility

### PR DESCRIPTION
**Pull request by adityaamoghe**
Added feature that makes sure that password and confirm password are the same.

### CODE REVIEW

File: app/public/javascripts/passValid.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+function ValidatePassword (){
+    const myForm = document.getElementById('myForm');
+    const password = myForm.elements['password'].value;
```
Comment (rcherukuru2000) [inline app/public/javascripts/passValid.js:3]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-fa21-group17/cse110-fa21-group17/pullRequest?prid=8443809)
```suggestion
    const password = myForm.elements["password"].value;
```

File: app/public/javascripts/passValid.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+function ValidatePassword (){
+    const myForm = document.getElementById('myForm');
```
Comment (rcherukuru2000) [inline app/public/javascripts/passValid.js:2]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-fa21-group17/cse110-fa21-group17/pullRequest?prid=8443809)
```suggestion
    const myForm = document.getElementById("myForm");
```

**Review by ThomasLiARDJAVA — APPROVED** [review]
LGTM!
Just make sure to modify on the same branch and same PR next time. No need to create a new branch and new PR whenever there are changes needed.

File: app/public/javascripts/passValid.js (hunk 1 above)
Comment (rcherukuru2000) [inline app/public/javascripts/passValid.js:3]: Codacy has a fix for the issue: [Expected indentation of 2 spaces but found 4.](https://app.codacy.com/gh/cse110-fa21-group17/cse110-fa21-group17/pullRequest?prid=8443809)
```suggestion
  const password = myForm.elements['password'].value;
```

File: app/public/javascripts/passValid.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    const password = myForm.elements['password'].value;
+    const confirm_password = myForm.elements['confirm_password'].value;
+
+    if(password !== confirm_password){
```
Comment (rcherukuru2000) [inline app/public/javascripts/passValid.js:6]: Codacy has a fix for the issue: [Expected indentation of 2 spaces but found 4.](https://app.codacy.com/gh/cse110-fa21-group17/cse110-fa21-group17/pullRequest?prid=8443809)
```suggestion
  if(password !== confirm_password){
```

File: app/public/javascripts/passValid.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+        alert("Please make sure passwords match.");
+        return false;
+    }
```
Comment (rcherukuru2000) [inline app/public/javascripts/passValid.js:9]: Codacy has a fix for the issue: [Expected indentation of 2 spaces but found 4.](https://app.codacy.com/gh/cse110-fa21-group17/cse110-fa21-group17/pullRequest?prid=8443809)
```suggestion
  }
```

File: app/public/javascripts/passValid.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+function ValidatePassword (){
+    const myForm = document.getElementById('myForm');
+    const password = myForm.elements['password'].value;
+    const confirm_password = myForm.elements['confirm_password'].value;
```
Comment (rcherukuru2000) [inline app/public/javascripts/passValid.js:4]: Codacy has a fix for the issue: [Expected indentation of 2 spaces but found 4.](https://app.codacy.com/gh/cse110-fa21-group17/cse110-fa21-group17/pullRequest?prid=8443809)
```suggestion
  const confirm_password = myForm.elements['confirm_password'].value;
```

File: app/public/javascripts/passValid.js (hunk 4 above)
Comment (rcherukuru2000) [inline app/public/javascripts/passValid.js:9]: Codacy has a fix for the issue: [Closing curly brace does not appear on the same line as the subsequent block.](https://app.codacy.com/gh/cse110-fa21-group17/cse110-fa21-group17/pullRequest?prid=8443809)
```suggestion
    } else{
```

File: app/public/javascripts/passValid.js [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+
+    if(password !== confirm_password){
+        alert("Please make sure passwords match.");
```
Comment (rcherukuru2000) [inline app/public/javascripts/passValid.js:7]: Codacy has a fix for the issue: [Expected indentation of 4 spaces but found 8.](https://app.codacy.com/gh/cse110-fa21-group17/cse110-fa21-group17/pullRequest?prid=8443809)
```suggestion
    alert("Please make sure passwords match.");
```

File: app/public/javascripts/passValid.js [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+        return false;
+    }
+    else{
```
Comment (rcherukuru2000) [inline app/public/javascripts/passValid.js:10]: Codacy has a fix for the issue: [Expected indentation of 2 spaces but found 4.](https://app.codacy.com/gh/cse110-fa21-group17/cse110-fa21-group17/pullRequest?prid=8443809)
```suggestion
  else{
```

File: app/public/javascripts/passValid.js [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+        return true;
```
Comment (rcherukuru2000) [inline app/public/javascripts/passValid.js:11]: Codacy has a fix for the issue: [Expected indentation of 4 spaces but found 8.](https://app.codacy.com/gh/cse110-fa21-group17/cse110-fa21-group17/pullRequest?prid=8443809)
```suggestion
    return true;
```

File: app/public/javascripts/passValid.js [hunk 9]
```diff
@@ -0,0 +1,13 @@
+function ValidatePassword (){
+    const myForm = document.getElementById('myForm');
+  const password = myForm.elements['password'].value;
+  const confirm_password = myForm.elements['confirm_password'].value;
+
+  if(password !== confirm_password){
+    alert("Please make sure passwords match.");
+        return false;
+  }
+  else{
```
Comment (rcherukuru2000) [inline app/public/javascripts/passValid.js:10]: Codacy has a fix for the issue: [Expected space(s) after "else".](https://app.codacy.com/gh/cse110-fa21-group17/cse110-fa21-group17/pullRequest?prid=8443809)
```suggestion
    else {
```

File: app/public/javascripts/passValid.js [hunk 10]
```diff
@@ -0,0 +1,13 @@
+function ValidatePassword (){
+    const myForm = document.getElementById('myForm');
+  const password = myForm.elements['password'].value;
+  const confirm_password = myForm.elements['confirm_password'].value;
+
+  if(password !== confirm_password){
```
Comment (rcherukuru2000) [inline app/public/javascripts/passValid.js:6]: Codacy has a fix for the issue: [Expected space(s) after "if".](https://app.codacy.com/gh/cse110-fa21-group17/cse110-fa21-group17/pullRequest?prid=8443809)
```suggestion
    if (password !== confirm_password){
```

File: app/public/javascripts/passValid.js [hunk 11]
```diff
@@ -0,0 +1,13 @@
...
+    alert("Please make sure passwords match.");
+        return false;
+  }
```
Comment (rcherukuru2000) [inline app/public/javascripts/passValid.js:9]: Codacy has a fix for the issue: [Closing curly brace does not appear on the same line as the subsequent block.](https://app.codacy.com/gh/cse110-fa21-group17/cse110-fa21-group17/pullRequest?prid=8443809)
```suggestion
    } else{
```

File: app/public/javascripts/passValid.js [hunk 12]
```diff
@@ -0,0 +1,13 @@
+function ValidatePassword (){
```
Comment (rcherukuru2000) [inline app/public/javascripts/passValid.js:1]: Codacy has a fix for the issue: [Unexpected space before function parentheses.](https://app.codacy.com/gh/cse110-fa21-group17/cse110-fa21-group17/pullRequest?prid=8443809)
```suggestion
function ValidatePassword(){
```

File: app/public/javascripts/passValid.js [hunk 13]
```diff
@@ -0,0 +1,13 @@
...
+  const password = myForm.elements['password'].value;
+  const confirm_password = myForm.elements['confirm_password'].value;
+
+  if(password !== confirm_password){
+    alert("Please make sure passwords match.");
+        return false;
+  }
+  else{
+    return true;
+    }
```
Comment (rcherukuru2000) [inline app/public/javascripts/passValid.js:12]: Codacy has a fix for the issue: [Expected indentation of 2 spaces but found 4.](https://app.codacy.com/gh/cse110-fa21-group17/cse110-fa21-group17/pullRequest?prid=8443809)
```suggestion
  }
```

File: app/routes/index.js [hunk 14]
```diff
@@ -33,7 +33,7 @@ router.post('/signup', async function(req, res, next){
       return res.status(401)
           .json({status: 'failed', message: 'Another account using this email was found'});
     }
-
+
```
Comment (rcherukuru2000) [inline app/routes/index.js:36]: Codacy has a fix for the issue: [Trailing spaces not allowed.](https://app.codacy.com/gh/cse110-fa21-group17/cse110-fa21-group17/pullRequest?prid=8443809)
```suggestion

```

File: app/routes/index.js [hunk 15]
```diff
@@ -6,8 +6,8 @@ const usersModel = require('../database/models/usersModel');
 
 /* GET home page. */
 router.get('/', function(req, res, next) {
-  res.render('index', { title: 'Express' });
-});
+  res.render('index', { title: 'Express' });
```
Comment (rcherukuru2000) [inline app/routes/index.js:9]: Codacy has a fix for the issue: [Trailing spaces not allowed.](https://app.codacy.com/gh/cse110-fa21-group17/cse110-fa21-group17/pullRequest?prid=8443809)
```suggestion
  res.render('index', { title: 'Express' });
```

File: app/public/javascripts/passValid.js [hunk 16]
```diff
@@ -0,0 +1,13 @@
+function ValidatePassword (){
+    const myForm = document.getElementById('myForm');
+  const password = myForm.elements['password'].value;
+  const confirm_password = myForm.elements['confirm_password'].value;
+
+  if(password !== confirm_password){
+    alert("Please make sure passwords match.");
+        return false;
```
Comment (rcherukuru2000) [inline app/public/javascripts/passValid.js:8]: Codacy has a fix for the issue: [Expected indentation of 4 spaces but found 8.](https://app.codacy.com/gh/cse110-fa21-group17/cse110-fa21-group17/pullRequest?prid=8443809)
```suggestion
    return false;
```

**Pull request merged**